docker-compose.dev-full.yml

```diff
@@ -129,6 +129,8 @@ services:
         extends:
             file: docker-compose.base.yml
             service: capture
+        ports:
```

Comment:
I don't follow this PR tbh. The caddy stuff already takes care of all of this without exposing ports. I can't find any path that would send it to port 3000?

Reply:
Not to mention this will clash with a bunch of other services so I think this doesn't solve what you think it does

Reply:
this fixes localhost:8000 for me with no clashes

because of this line https://github.com/PostHog/posthog/blob/c54ce4ab1212d653fff48f98b060e9634c851a4b/posthog/urls.py#L252

Reply:
if you use localhost:8000 (which is still the default) then you're not hitting caddy, no?